ruff.toml

```diff
@@ -26,6 +26,7 @@ exclude = [
     "node_modules",
     "site-packages",
     "venv",
+    "**test**"
```

Comment:
One thing too, since there were many, many errors in the test files was I just excluded all of them for the sake of time. If someone wants to go back and fix those as well by all means :)